### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by inson1** [dylanaraps/neofetch#2453]
Last commit was in December 2021. And last version is from 2020.

**Comment by nonetrix** [comment]
Also wondering the same thing, it's pretty much feature complete but improvements could still be made

**Comment by inson1** [comment]
@dylanaraps Hi. Is this project dead?

**Comment by inson1** [comment]
little bit longer hiatus

**Comment by nonetrix** [comment]
It also seems all of his websites are down, also be careful because it seems it's being domain squatted or whatever to promote phishing. Likely not the fault of the original maintainer so don't blame him of course, likely his domain expired and someone malicious bought it up

**Comment by polluks** [comment]
Take a look at this fork https://github.com/Un1q32/pfetch

**Comment by nonetrix** [comment]
That is pfetch which is completely separate pretty sure

**Comment by Desyncfy** [comment]
yeah seems pretty dead I'd just use hyfetch because it continues to get updates.
